packages/twenty-front/src/modules/workflow/components/RightDrawerWorkflowEditStepContent.tsx

```diff
@@ -0,0 +1,93 @@
+import { RightDrawerWorkflowEditStepContentAction } from '@/workflow/components/RightDrawerWorkflowEditStepContentAction';
+import { RightDrawerWorkflowEditStepContentTrigger } from '@/workflow/components/RightDrawerWorkflowEditStepContentTrigger';
+import { TRIGGER_STEP_ID } from '@/workflow/constants/TriggerStepId';
+import { useUpdateWorkflowVersionStep } from '@/workflow/hooks/useUpdateWorkflowVersionStep';
+import { useUpdateWorkflowVersionTrigger } from '@/workflow/hooks/useUpdateWorkflowVersionTrigger';
+import { showPageWorkflowSelectedNodeState } from '@/workflow/states/showPageWorkflowSelectedNodeState';
+import { WorkflowWithCurrentVersion } from '@/workflow/types/Workflow';
+import { findStepPositionOrThrow } from '@/workflow/utils/findStepPositionOrThrow';
+import { useRecoilValue } from 'recoil';
+import { isDefined } from 'twenty-ui';
+
+const getStepDefinition = ({
+  stepId,
+  workflow,
+}: {
+  stepId: string;
+  workflow: WorkflowWithCurrentVersion;
+}) => {
+  const currentVersion = workflow.currentVersion;
+  if (!isDefined(currentVersion)) {
+    return undefined;
+  }
+
+  if (stepId === TRIGGER_STEP_ID) {
+    if (!isDefined(currentVersion.trigger)) {
+      return undefined;
+    }
+
+    return {
+      type: 'trigger',
+      definition: currentVersion.trigger,
+    } as const;
+  }
+
+  if (!isDefined(currentVersion.steps)) {
+    return undefined;
+  }
+
+  const selectedNodePosition = findStepPositionOrThrow({
+    steps: currentVersion.steps,
+    stepId: stepId,
+  });
+
+  return {
+    type: 'action',
+    definition: selectedNodePosition.steps[selectedNodePosition.index],
+  } as const;
+};
+
+export const RightDrawerWorkflowEditStepContent = ({
```

Comment:
I see you have several components with the prefix `RightDrawer` or `ShowPage`. Most of components do not need to know what is the parent. Here for example, we do not necessarily need to use it in a right drawer.

Please update your child component names and keep the prefix for the first parent.

Reply:
I renamed most of the components. Only four components remain with the prefix RightDrawer. I kept the prefix for the entry point components and their immediate unique child. The two components ending with `Content` contain most of the logic and are displayed when the workflow has been successfully loaded. They can't exist on their own.